src/20240229_Multi_AssignEmissionAdminEthereumArbitirumAndOptimism/AssignEmissionAdminEthereumArbitirumAndOptimism.md

```diff
@@ -22,37 +23,37 @@ The OP and ARB incentives are being distributed with the intent of migrating USD
 
 ## Specification
 
-The `EmissionManager`, a governance controlled function, assigns the `Emission_Admin` role for a specified token on the respective network.
+The `setEmissionAdmin`, a governance controlled function, assigns the `EMISSION_MANAGER` role for a specified token on the respective network.
```

Comment:
i commented that you changed the casing in every sentence, now you just replaces everything with `EMISSION_MANAGER`which is wrong. E.g. here should be `EMISSION_ADMIN` like it was before.